janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java

```diff
@@ -0,0 +1,116 @@
+// limitations under the License.
+
+package org.janusgraph.hadoop;
+
+import org.janusgraph.core.Cardinality;
+import org.janusgraph.core.JanusGraphVertex;
+import org.janusgraph.diskstorage.configuration.WriteConfiguration;
+import org.janusgraph.example.GraphOfTheGodsFactory;
+import org.janusgraph.graphdb.JanusGraphBaseTest;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.apache.tinkerpop.gremlin.spark.process.computer.SparkGraphComputer;
+import org.apache.tinkerpop.gremlin.structure.Direction;
+import org.apache.tinkerpop.gremlin.structure.Edge;
+import org.apache.tinkerpop.gremlin.structure.Graph;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+import org.apache.tinkerpop.gremlin.structure.util.GraphFactory;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public abstract class AbstractInputFormatIT extends JanusGraphBaseTest {
+
+
+    @Test
+    public void testReadGraphOfTheGods() throws Exception {
+        GraphOfTheGodsFactory.load(graph, null, true);
+        assertEquals(12L, (long) graph.traversal().V().count().next());
+        Graph g = getGraph();
+        GraphTraversalSource t = g.traversal(GraphTraversalSource.computer(SparkGraphComputer.class));
+        assertEquals(12L, (long) t.V().count().next());
+    }
+
+    @Test
+    public void testReadWideVertexWithManyProperties() throws Exception {
+        int numProps = 1 << 16;
+
+        long numV  = 1;
+        mgmt.makePropertyKey("p").cardinality(Cardinality.LIST).dataType(Integer.class).make();
+        mgmt.commit();
+        finishSchema();
+
+        for (int j = 0; j < numV; j++) {
+            Vertex v = graph.addVertex();
+            for (int i = 0; i < numProps; i++) {
+                v.property("p", i);
+            }
+        }
+        graph.tx().commit();
+
+        assertEquals(numV, (long) graph.traversal().V().count().next());
+        Map<String, Object> propertiesOnVertex = graph.traversal().V().valueMap().next();
+        List<?> valuesOnP = (List)propertiesOnVertex.values().iterator().next();
+        assertEquals(numProps, valuesOnP.size());
+        Graph g = getGraph(); 
+        GraphTraversalSource t = g.traversal(GraphTraversalSource.computer(SparkGraphComputer.class));
+        assertEquals(numV, (long) t.V().count().next());
+        propertiesOnVertex = t.V().valueMap().next();
+        valuesOnP = (List)propertiesOnVertex.values().iterator().next();
+        assertEquals(numProps, valuesOnP.size());
```

Comment:
This was the version that was in CassandraInputFormatIT and was not modified in this PR. I think changing it would be out of scope here but maybe create an issue for it?